.github/workflows/release-drafter.yaml

```diff
@@ -0,0 +1,19 @@
+name: Release Drafter
+
+on:
+  push:
+    branches:
+      - main
+
+permissions:
+  contents: read
+
```

Comment:
Could we add a short comment here describing what the release-drafter action does? Something like:
```suggestion
# the release-drafter action will compile release notes as pull-requests are merged. A draft release is kept up-to-date listing the changes,